libkodimote/tvshows.h

```diff
@@ -43,6 +43,7 @@ class TvShows : public KodiLibrary
 
     ThumbnailFormat thumbnailFormat() const { return ThumbnailFormatLandscape; }
     bool allowWatchedFilter() { return true; }
+    QString watchedFilterSetting() { return "showWatchedTvShows"; }
```

Comment:
I don't particularly like the fact that we have to add setting names to the models...

How about this:
* We add a QVariantMap typed setting called "watchedFilterSettings" and another called "sortingSettings".
* To KodiModel we add a read-only property "modelName" returning metaObject()->className()

Then in the BrowserPage we can store sorting/filter settings somehow like:
 settings.watchedFilterSettings[model.modelName] = showWatched;

You think that would work?